pal/audio/web/player-web.ts

```diff
@@ -95,7 +95,10 @@ export class AudioContextAgent {
                 resolve();
                 return;
             }
-            context.resume().catch((e) => { debug.warn('runContext error', e); });
+            // only state is suspended can involve resume function
+            if (context.state === 'suspended') {
+                context.resume().catch((e) => { debug.warn('runContext error', e); });
```

Comment:
After the logic change, will low-end devices be able to play normally?

Reply:
I have test ios 15.7 ios 14.8 and ios 17.5.1, three version of iphone.